### PR TITLE
Contributing a GetFileExtension custom skill 

### PR DESCRIPTION
**Pull request by rob-derosa**
## Purpose

Allows users to index and filter by document file extension

## Does this introduce a breaking change?

```
[ ] Yes
[ X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
*  Follow the readme

### CODE REVIEW

File: Utils/GetFileExtension/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+---
+
+# GetFileExtension
+
+This custom skill returns the document's file extension and the file name without extension to be indexed accordingly.
+
+## Requirements
+
+These skills have no additional requirements than the ones described in [the root `README.md` file](../../README.md).
+
```
Comment (bleroy) [inline Utils/GetFileExtension/README.md]: There should be a deployment section here (see other skills and use them as a template).

Comment (rob-derosa) [reply Utils/GetFileExtension/README.md]: Added :)

**Comment by bleroy** [comment]
Cool, thanks for the fixes. Would you mind also updating the top-level readme so it has a pointer to your skill? I'll wait a bit for this before I merge. Otherwise this looks all good.

**Comment by rob-derosa** [comment]
Yeah, sorry about that.

**Comment by bleroy** [comment]
Thanks!

**Pull request merged**